frameworks/cassandra/tests/test_sanity.py

```diff
@@ -96,3 +99,25 @@ def expected_metrics_exist(emitted_metrics):
         config.DEFAULT_CASSANDRA_TIMEOUT,
         expected_metrics_exist,
     )
+
+
+@pytest.mark.sanity
+def test_custom_jmx_port():
+    expected_port = "-Dcassandra.jmx.local.port=7200"
+
+    new_config = {"cassandra": {"jmx_port": 7200}}
+
+    sdk_service.update_configuration(
+        config.PACKAGE_NAME,
+        config.get_foldered_service_name(),
+        new_config,
+        config.DEFAULT_TASK_COUNT,
+    )
+
+    sdk_plan.wait_for_completed_deployment(config.get_foldered_service_name())
+
+    tasks = sdk_tasks.get_service_tasks(config.get_foldered_service_name(), "node")
+
+    for task in tasks:
+        _, stdout, _ = sdk_cmd.run_cli("task log --lines=1000 {}".format(task.id))
```

Comment:
It would be better to check if this port is actually used by the process instead of relying on the log output.

Reply:
good call. Made the change in 515e77a